src/core/lib/gprpp/inlined_vector.h

```diff
@@ -125,6 +125,16 @@ class InlinedVector {
     }
   }
 
+  void resize(size_t new_size) {
+    if (new_size == size_) return;
+    while (new_size > size_) {
+      emplace_back();
+    }
+    while (new_size < size_) {
+      data()[size_-- - 1].~T();
```

Comment:
Suggest changing `size_-- -1` to `--size_`.

Or better yet, just use `pop_back()` here.

Reply:
Done.